core-bundle/src/Twig/Interop/PhpTemplateProxyNode.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Twig\Interop;
+
+use Twig\Compiler;
+use Twig\Node\Node;
+
+class PhpTemplateProxyNode extends Node
+{
+    public function __construct(string $extensionName)
+    {
+        parent::__construct([], [
+            'extension_name' => $extensionName,
+        ]);
+    }
+
+    public function compile(Compiler $compiler): void
+    {
+        /*
+         * echo $this->extensions["Contao\\…\\ContaoExtension"]->renderLegacyTemplate(
+         *     $this->getTemplateName(),
+         *     array_map(
+         *         static function(callable $block) use ($context): string {
+         *             if ($this->env->isDebug()) { ob_start(); } else { ob_start(function () { return ''; }); }
+         *             try { $block($context); return ob_get_contents(); } finally { ob_end_clean(); }
+         *         }, $blocks
+         *     ), $context
+         * );
+         */
+
+        $compiler
+            ->write('echo $this->extensions[')
+            ->repr($this->getAttribute('extension_name'))
+            ->raw(']->renderLegacyTemplate('."\n")
+            ->indent()
+            ->write('$this->getTemplateName(),'."\n")
+            ->write('array_map('."\n")
+            ->indent()
+            ->write('static function(callable $block) use ($context): string {'."\n")
```

Comment:
```suggestion
            ->write('function(callable $block) use ($context): string {'."\n")
```
this must not be static IMO, otherwise the `$this->env->isDebug()` triggers an error.

Reply:
Good catch, fixed in 816a7c247e13d767bb758f633591f15f8ddcb9a0